new/LIT7293MartyrMb.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" type="work" xml:lang="en" xml:id="LIT7293MartyrMb">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ስምዕ፡ ዘቅዱስ፡ መርቆርዮስ፡</title>
+                <title xml:lang="en" corresp="#t1">The Martyrdom of Mercurios (recension b)</title>
```

Comment:
If the first is α, should this not be β? Or should they all be with Latin lower-case letters?

Reply:
changed to Latin lower-case letters.